sklearn_pandas/dataframe_mapper.py

```diff
@@ -97,8 +98,8 @@ def __init__(self, features, default=False, sparse=False, df_out=False,
                     numpy array. Defaults to ``False``.
 
         """
-        self.features = features
-        self.built_features = None
+        self._drop_columns = self._drop_columns(features)
```

Comment:
Instead of adding it as a feature, we can add an additional variable in the constructor that take a list of columns to drop. Something like this:

`def __init__(self, features, default=False, sparse=False, df_out=False, drop_column=None):`